test/e2e/placeholder_secrets_simple_case_test.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/ghodss/yaml"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// TestPlaceholderSuccessful : TODO - we have a more complicated test that should make this one redundant; move things from here into unit tests and/or remove this file
+func TestPlaceholderSuccessful(t *testing.T) {
+	env := BuildEnv(t)
+	logger := Logger{}
+	kapp := Kapp{t, env.Namespace, logger}
+	kubectl := Kubectl{t, env.Namespace, logger}
+
+	yaml1 := `
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sg-test1
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sg-test2
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sg-test3
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret
+  namespace: sg-test1
+type: kubernetes.io/dockerconfigjson
+stringData:
+  .dockerconfigjson: |
+    {
+      "auths": {
+        "www.sg-test1-server.com": {
+          "username": "sg-test1-secret-user",
+          "password": "sg-test1-secret-password",
+          "auth": "sgtest1-notbase64"
+        }
+      }
+    }
+---
+apiVersion: secretgen.k14s.io/v1alpha1
+kind: SecretExport
+metadata:
+  name: secret
+  namespace: sg-test1
+spec:
+  toNamespaces:
+  - sg-test2
+  - sg-test3
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+    secretgen.carvel.dev/image-pull-secret: ""
+  name: secret
```

Comment:
lets rename this secret to something like placeholder-secret or dst-secret. otherwise gets a little confusing what with all of them being same name.